assets/src/amp-validation/amp-validated-url-post-edit-screen.js

```diff
@@ -376,3 +377,27 @@ const handleBulkActions = () => {
 		} );
 	} );
 };
+
+/**
+ * Set up stylesheet metabox.
+ */
+const setupStylesheetsMetabox = () => {
+	const metabox = document.getElementById( 'amp_stylesheets' );
+
+	for ( const toggleStylesheetDetailsButton of metabox.querySelectorAll( '.toggle-stylesheet-details' ) ) {
+		const row = toggleStylesheetDetailsButton.closest( 'tr' );
+		toggleStylesheetDetailsButton.addEventListener( 'click', () => {
+			row.classList.toggle( 'expanded' );
+		} );
+	}
+
+	for ( const stylesheetDetailsElements of metabox.querySelectorAll( '.stylesheet-details' ) ) {
+		const shakenStylesheetContainer = stylesheetDetailsElements.querySelector( '.shaken-stylesheet' );
+		const showRemovedStylesCheckbox = stylesheetDetailsElements.querySelector( '.show-removed-styles' );
+		if ( showRemovedStylesCheckbox ) {
+			showRemovedStylesCheckbox.addEventListener( 'click', () => {
+				shakenStylesheetContainer.classList.toggle( 'removed-styles-shown', showRemovedStylesCheckbox.checked );
```

Comment:
This doesn't look to be working in my local:

![div-stylesheet](https://user-images.githubusercontent.com/4063887/73053535-93206880-3e4d-11ea-8453-f156cb41e418.gif)

I didn't see any console error.

Reply:
This is working fine now with a hard refresh.